logdna/client.go

```diff
@@ -128,29 +220,41 @@ func MakeRequestView(c *Client, url string, urlsuffix string, method string, pay
 
 // CreateAlert creates a Preset Alert with the provided payload
 func (c *Client) CreateAlert(url string, payload ViewPayload) (string, error) {
-	result, err := MakeRequestAlert(c, url, "/v1/config/alert", "POST", payload)
+	result, err := MakeRequestAlert(c, url, "/v1/config/presetalert", "POST", payload)
```

Comment:
current CI errors have to do with the alert => presetalert update not having gone to prod yet